Makefile

```diff
@@ -31,8 +31,9 @@ test-efm:
 
 test-syntax:
 	$(SILENT) vim -T dumb -E \
+	        -c 'source test/functions.vim' \
 		-c 'source test/run_test.vim' \
-		-c 'call RunTestSyntax()' > test-syntax.log
+		-c 'call RunTestSyntax()' | tr -d '[
]' > test-syntax.log
```

Comment:
Were you able to test this on your system? I'm guessing this is related with TOhtml, but I can't seem to find any option to overcome this behavior.